src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -150,6 +160,32 @@ export class MarkdownTextTransformer {
 		}
 	}
 
+	/**
+	 * Instantiate modal for use as callout when inserting Link or Image into markdown.
+	 * @param calloutStyle Style of callout passed in to determine which callout is rendered
+	 */
+	private async createCallout(type: MarkdownButtonType, triggerElement: HTMLElement): Promise<string> {
+		const posX = triggerElement.getBoundingClientRect().left;
+		const posY = triggerElement.getBoundingClientRect().top;
+
+		let calloutString: unknown = type === MarkdownButtonType.IMAGE_PREVIEW ? 'IMAGE' : 'LINK';
+		let calloutType = (calloutString as ICalloutType);
+
+		let title = type === MarkdownButtonType.IMAGE_PREVIEW ? this.insertImageHeading : this.insertLinkHeading;
+
+		if (!this._callout) {
+			// Offset for Mac clients
```

Comment:
Why? If there's edge case stuff like this please make sure to always document why it's needed

Reply:
For now, I removed all dependencies on the `.mac` body class.